.rubocop.yml

```diff
@@ -13,6 +14,7 @@ AllCops:
     - Rakefile
     - Guardfile
     - Vagrantfile
+    - lib/puppet/jenkins/okjson.rb
```

Comment:
@bastelfreak I don't think modulesync_config currently supports adding per module excludes like this?

Reply:
It doesn't. I try to avoid this and and disable the 'broken' cops in the file itself.

Reply:
That would make periodically syncing from the upstream a bit more of a pain I guess.  I suppose we could just disable all applicable cops in 20 or so lines directly at the top of the file?  We'll need to decide when we get to the point we can actually try the modulesync again.